src/views/Event/index.js

```diff
@@ -43,8 +43,7 @@ class EventPage extends React.Component {
             let formData = mapAPIDataToUIFormat(event)
             formData.id = undefined
             delete formData.id
-
-            replaceData(formData)
+            replaceData(formData, constants.PUBLICATION_STATUS.PUBLIC)
```

Comment:
Actually the event has two statuses, `event_status` and `publication_status`.

To fix #279, we should set `event_status` to `EventScheduled`. I think `publication_status` doesn't need to be checked here, because it is allowed to be either draft or public.

Reply:
fixed in a new forced push. Default status to `public`